sos/report/plugins/networking.py

```diff
@@ -246,14 +242,6 @@ def setup(self):
                     ], devices=_devs['ethernet'], priority=50, subdir=_subdir)
 
         self.add_cmd_tags({
-            "ethtool [^-].*": "ethtool",
```

Comment:
These tags were already done from lines 49-51.

Reply:
Good point!

For the sake of code "structure" or clarity, does it make sense to *move* the lines 49-51 here, like:

        for opt in self.ethtool_shortopts:
            self.add_cmd_tags({
                'ethtool -%s .*' % opt: 'ethool_%s' % opt
            })
        self.add_cmd_tags({
            "ip -d address": "ip_addr",
            ..

? To have all tags on one place.

Reply:
Well, the

    "ethtool [^-].*": "ethtool",

should remain there - it is not covered by the loop on lines 49-51 and we call `ethtool %(dev)s` on line 144.

Reply:
Good catch!
`"ethtool [^-].*": "ethtool",` is back now.

Reply:
Also moved the `add_cmd_tags` together but moved into a new function on account of too-many-locals.